packages/flutter/lib/src/material/banner.dart

```diff
@@ -138,18 +179,101 @@ class MaterialBanner extends StatelessWidget {
   /// Defaults to [OverflowBarAlignment.end].
   final OverflowBarAlignment overflowAlignment;
 
+  /// The animation driving the entrance and exit of the material banner.
+  final Animation<double>? animation;
```

Comment:
This should document what happens if this is null/default behavior.
```suggestion
  /// The animation driving the entrance and exit of the material banner when presented by the [ScaffoldMessenger].
  final Animation<double>? animation;
```